app/api/routers/breeze_buddy/user_accounts/handlers.py

```
@@ -0,0 +1,430 @@
+            merchant_identifiers=user_data.merchant_identifiers,
+            is_active=user_data.is_active,
+            owner_id=current_user.id,
+        )
+
+        if not user:
+            raise HTTPException(status_code=500, detail="Failed to create user")
+
+        return user
+
+    except asyncpg.UniqueViolationError:
+        raise HTTPException(status_code=409, detail=f"ID or username already exists")
```

Ruff (0.15.4)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)